README.md

```diff
@@ -68,7 +68,8 @@ Creating a new release
 
 1. Bump the version number in `package.json` and `src/linter.ts`
 2. Add release notes in `CHANGELOG.md`
-3. `yarn verify` to build the latest sources from a clean state
 4. Commit with message `Prepare release <version>`
-5. Run `npm publish`
-6. Create a git tag for the new release and push it ([see existing tags here](https://github.com/palantir/tslint/tags))
+5. Push your branch to GitHub and make a PR
+6. Once your PR is merged, wait for the tests to pass on CircleCI for develop
+7. Create a "Release" on GitHub with the proper tag version and notes from the changelog
+8. Run 'yarn run publish:local 4.1.0' (replace with the appropriate tag name for the release)
```

Comment:
I believe that's only needed if you're trying to pass dash options (like passing `-o` to your script or something like that)